### PR TITLE
Add Vietnamese translation for Sandboxie-Plus

### PR DESCRIPTION
**Pull request by GunGunGun**
Please check and accept my pull request, 100% done, for now.

Please let me know if there's any file that I need to translate, I only translated sandman_en.ts.

Also can I get supporter certificate ? 

Thanks for continuing this awesome piece of software, hopefully I will be able to contribute more in the future.

> Thank you for your contribution to the Sandboxie repository.
> 
> In order to reduce the risk of accidental merges, it's highly recommended for beginners to open a new Pull Request in draft state by clicking the button "Create Draft Pull Request", instead of using the default "Create Pull Request" option. 
> 
> In addition, you can convert an existing pull request to a draft by clicking the "Convert to draft" link in the right sidebar under "Reviewers".
> 
> All new translators are encouraged to look at the "Localization notes and tips" before creating a pull request: https://git.io/J9G19

### CODE REVIEW

**Comment by isaak654** [comment]
Other dependencies to translate / add:

- [ ] 1. <a href="https://github.com/sandboxie-plus/Sandboxie/blob/master/Installer/Languages.iss">Here</a> you can find the location of the `Sandboxie Plus` installer languages file where you need to translate the english lines in the [CustomMessages] section

- [ ] 2. <a href="https://github.com/sandboxie-plus/Sandboxie/blob/master/Installer/Sandboxie-Plus.iss#L206-L224">Here</a> you can find the location of the `Sandboxie Plus` installer script where you need to add the language value, for example: `'vietnamese': Result := 'vi';`

- [ ] 3. Upload the [Vietnamese unofficial language for InnoSetup](https://github.com/jrsoftware/issrc/blob/main/Files/Languages/Unofficial/Vietnamese.isl) in our [custom ISL folder](https://github.com/sandboxie-plus/Sandboxie/tree/master/Installer/isl)

- [ ] 4. Add `Name: "vietnamese"; MessagesFile: "isl\Vietnamese.isl"` in the [Languages] section of [Languages.iss](https://github.com/sandboxie-plus/Sandboxie/blob/master/Installer/Languages.iss)

File: SandboxiePlus/SandMan/sandman_vi.ts [hunk 1]
```diff
@@ -0,0 +1,5904 @@
...
+    <message>
+        <location filename="Forms/OptionsWindow.ui" line="3662"/>
+        <source>Add Option</source>
+        <translation type="unfinished">Thêm tùy chọn</translation>
+    </message>
+    <message>
+        <location filename="Forms/OptionsWindow.ui" line="3676"/>
+        <source>Here you can configure advanced per process options to improve compatibility and/or customize sandboxing behavior.</source>
+        <oldsource>Here you can configure advanced per process options to improve compatibility and/or customize sand boxing behavior.</oldsource>
+        <translation type="unfinished">Tại đây, bạn có thể định cấu hình các tùy chọn nâng cao cho mỗi quy trình để cải thiện khả năng tương thích và / hoặc tùy chỉnh hành vi Sandbox. </translation>
```
Comment (isaak654) [inline SandboxiePlus/SandMan/sandman_vi.ts:5008]:
```suggestion
        <translation type="unfinished">Tại đây, bạn có thể định cấu hình các tùy chọn nâng cao cho mỗi quy trình để cải thiện khả năng tương thích và / hoặc tùy chỉnh hành vi Sandbox.</translation>
```

The extra space is not needed.

**Review by isaak654 — CHANGES_REQUESTED** [review]

File: SandboxiePlus/SandMan/sandman_vi.ts [hunk 2]
```diff
@@ -0,0 +1,5904 @@
...
+        <source>Use Dark Mode</source>
+        <translation>Sử dụng Chế độ tối</translation>
+    </message>
+</context>
+<context>
+    <name>CWFPPage</name>
+    <message>
+        <location filename="Wizards/SetupWizard.cpp" line="454"/>
+        <source>Configure &lt;b&gt;Sandboxie-Plus&lt;/b&gt; network filtering</source>
+        <translation>Định cấu hình lọc mạng của &lt;b&gt;Sandboxie-Plus&lt;/b&gt; </translation>
```
Comment (isaak654) [inline SandboxiePlus/SandMan/sandman_vi.ts:3807]:
```suggestion
        <translation>Định cấu hình lọc mạng của &lt;b&gt;Sandboxie-Plus&lt;/b&gt;</translation>
```

File: SandboxiePlus/SandMan/sandman_vi.ts [hunk 3]
```diff
@@ -0,0 +1,5904 @@
...
+    </message>
+    <message>
+        <location filename="Wizards/SetupWizard.cpp" line="131"/>
+        <source>Select how you would like to use Sandboxie-Plus</source>
+        <translation>Chọn cách bạn muốn sử dụng Sandboxie-Plus</translation>
+    </message>
+    <message>
+        <location filename="Wizards/SetupWizard.cpp" line="134"/>
+        <source>&amp;Personally, for private non-commercial use</source>
+        <translation>&amp;Tôi muốn sử dụng cho riêng cá nhân.</translation>
```
Comment (isaak654) [inline SandboxiePlus/SandMan/sandman_vi.ts:114]:
```suggestion
        <translation>&amp;Tôi muốn sử dụng cho riêng cá nhân</translation>
```

**Review by isaak654 — CHANGES_REQUESTED** [review]

**Comment by GunGunGun** [comment]
@isaak654 Thanks, I'll try to translate *.iss and *.isl files later, hopefully they're not as important as sandman_*.ts.

**Comment by isaak654** [comment]
Personally, I would not approve this pull request without those files.
If you cannot find a way to include them in this pull request, they can be attached in any comment here and I'll make sure to insert them later.

**Comment by GunGunGun** [comment]
I've updated all the iss and isl files: https://github.com/sandboxie-plus/Sandboxie/pull/2439

**Review by isaak654 — APPROVED** [review]
Feel free to get in touch with @DavidXanatos by email for the contributor certificate.

**Pull request merged**